frontend/src/views/camp/CampProgram.vue

```diff
@@ -42,6 +62,17 @@ Show all activity schedule entries of a single period.
         </template>
       </template>
     </schedule-entries>
+    <v-snackbar v-model="showError" app :timeout="10000">
```

Comment:
does this clear the showError flag if the timeout passes?

Reply:
yes